fix: Apply button in filter dialog is confusing

zermelo-wisen opened the pull request:
Fixes #1347. 

Changed FilterMenu.vue:
- apply button title renamed from "Apply" to "Load"
- moved the button to the right of the filter select
- spacing between "Load" button and select on the left made consistent with Save Filter button and the input box on its left.

Before (Filter Menu storybook)
<img width="651" alt="Before" src="https://github.com/getappmap/appmap-js/assets/98758224/6d073c71-d401-4c2e-bf1a-1b9ce60d1934">

After (Filter Menu storybook)
<img width="621" alt="After" src="https://github.com/getappmap/appmap-js/assets/98758224/1bc84d6f-39e9-4444-a936-99d533d27539">

ahtrotta reviewed (changes requested):
Thanks for your PR! I like the change from `Apply` to `Load`, I think that's more clear. However, I think moving `Load` to the side of the select bar is a bit confusing because the `Delete`, `Set as Default`, and `Copy` buttons also apply to the selected filter. I'd like to see the following changes:

1. Move the `Load` button back to its original position.
2. Make the `Load` button disabled if the current filter state matches the selected saved filter state.
3. Please add a simple Cypress test in the `packages/components/tests/e2e/specs` folder.

zermelo-wisen commented:
- Made the changes above.
- Added a test file: `filterMenu.spec.js`.
- Fixed related tests in `appmap/viewFilter.spec.js` which are dependent on the order of disabled and enabled buttons, and "Load" (previously "Apply") button title.

ahtrotta reviewed (approved):
Looks good! Thank you!

appland-release commented:
:tada: This PR is included in version @appland/components-v2.60.1 :tada:

The release is available on:
- `@appland/components-v2.60.1`
- `@appland/components-v2.60.1`
- [GitHub release](https://github.com/getappmap/appmap-js/releases/tag/%40appland/components-v2.60.1)
- `@appland/components-v2.60.1`

Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot :package::rocket:
